less/alert.less

```diff
@@ -7,9 +7,58 @@
 	bottom: 0;
 
 	.alert {
+		&::before {
+			position: relative;
+			top: -15px;
+			left: -15px;
+			display: block;
+			height: 2px;
+			width: 0;
+			transition: inherit;
+		}
+
+		&.alert-info::before {
+			background-color: lighten(@material-info, 25%);
+		}
+
+		&.alert-warning::before {
+			background-color: lighten(@material-warning, 25%);
+		}
+
+		&.alert-success::before {
+			background-color: lighten(@material-success, 25%);
+		}
+
+		&.alert-danger::before {
+			background-color: lighten(@material-danger, 25%);
+		}
+
```

Comment:
:point_up: :point_down: Because your alerts are solid colours with white text, I inverted the transition to start light and end dark.